include/modem/lte_lc.h

```diff
@@ -182,12 +182,20 @@ int lte_lc_power_off(void);
  */
 int lte_lc_normal(void);
 
-/** @brief Function for requesting modem to go to or disable
- * power saving mode (PSM) with default settings defined in kconfig.
+/** @brief Function for setting modem PSM parameters (RPTAU and RAT)
+ * to be used when psm mode is subsequently enabled using `lte_lc_psm_req`.
  * For reference see 3GPP 27.007 Ch. 7.38.
  *
  * @return Zero on success or (negative) error code otherwise.
  */
+void lte_lc_psm_req_param_set(const char *req_rptau, const char *req_rat);
```

Comment:
The `req_` prefixes in the parameter names can be dropped

Reply:
I think we should return an error code from this function. It should fail if `NULL` is passed or a string of wrong length. It will eventually fail with a string of wrong length when PSM/eDRX is enabled, so better to give that feedback already in these functions

Reply:
Thanks for applying the change.
What I meant with the `_req` comment, was actually the prefix in the argument names, but removing it from the function name also makes sense.
The `r` in `rptau` and `rat` means `requested`, so the prefixes in argument names are redundant and should be removed.